snowplow_tracker/tracker_configuration.py

```diff
@@ -0,0 +1,64 @@
+# """
+#     tracker_configuration.py
+
+#     Copyright (c) 2013-2022 Snowplow Analytics Ltd. All rights reserved.
+
+#     This program is licensed to you under the Apache License Version 2.0,
+#     and you may not use this file except in compliance with the Apache License
+#     Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+#     http://www.apache.org/licenses/LICENSE-2.0.
+
+#     Unless required by applicable law or agreed to in writing,
+#     software distributed under the Apache License Version 2.0 is distributed on
+#     an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+#     express or implied. See the Apache License Version 2.0 for the specific
+#     language governing permissions and limitations there under.
+
+#     Authors: Jack Keene, Anuj More, Alex Dean, Fred Blundun, Paul Boocock
+#     Copyright: Copyright (c) 2013-2022 Snowplow Analytics Ltd
+#     License: Apache License Version 2.0
+# """
+
+from typing import Optional
+from snowplow_tracker.typing import JsonEncoderFunction
+
+class TrackerConfiguration(object):
+    def __init__(
+            self,
+            encode_base64: bool = None,
```

Comment:
```suggestion
            encode_base64: Optional[bool] = None,
```

Also in the comment below.